Dockerfile

```diff
@@ -63,10 +63,9 @@ ADD --chown=snyk:snyk . .
 RUN chmod 755 /srv/app && chmod 755 /srv/app/bin && chmod +x /srv/app/bin/start
 
 # This must be in the end for Red Hat Build Service
-RUN chown -R snyk:snyk .
```

Comment:
Why remove the chown?

Reply:
Isn't this duplicate from line 60?

Reply:
Looks intentionally added - https://github.com/snyk/kubernetes-monitor/commit/92bf2d925a8643288488a4a11dfede77599ce375